src/pages/App.tsx

```diff
@@ -54,117 +52,84 @@ interface IRoute {
   menuLink?: MENU_LINK //Bring it blank if it is same with path
   path: string | MENU_LINK
   component: React.FC
-  layout: React.FC
-  strict?: boolean
-  exact?: boolean
 }
 
 export default function App() {
   const chainId = useChainId()
   const routes: IRoute[] = [
     {
       path: MENU_LINK.dashboard,
-      component: Dashboard,
-      layout: Layout
+      component: Dashboard
```

Comment:
How are we recording which features are live for which chains? Is this inferred from the data in the SDK?

Reply:
as of now, its handled here `src/constants/accessPermissions.ts`